### PR TITLE
line follower java file added

### PR DESCRIPTION
**Pull request by Jayaneththi**

### CODE REVIEW

File: SDT2024 Team13/.gitignore
Comment (akilamadhufin) [inline SDT2024 Team13/.gitignore:1]: this code is working fine.

File: SDT2024 Team13/src/app/LineFollower.java
Comment (akilamadhufin) [inline SDT2024 Team13/src/app/LineFollower.java:1]: this code is working fine.

**Review by akilamadhufin — APPROVED** [review]
code seems okay.

**Review by Yashodha8890 — APPROVED** [review]
Seems like the code doesn't have any issue. Approving pull request.

**Comment by Jayaneththi** [comment]
line follower java code merged to master branch.

**Pull request merged**